tests/data/test_sql/test/incremental_query_v1/metadata.yaml

```diff
@@ -11,3 +11,4 @@ labels:
 scheduling:
   dag_name: "bqetl_events"
   depends_on_past: false
+  arguments: ["--append"]
```

Comment:
I was considering supporting only a subset of these arguments (based on the queries in telemetry-airflow it seems only a few of these options are used). That way these arguments could be provided without the `--`. But not sure if it's worth doing that.

Reply:
As it's now all `bq query` flags are supported by default.